ml-agents/mlagents/trainers/tf_policy.py

```diff
@@ -57,6 +57,7 @@ def __init__(self, seed, brain, trainer_parameters):
         self.brain = brain
         self.use_recurrent = trainer_parameters["use_recurrent"]
         self.memory_dict: Dict[str, np.ndarray] = {}
+        self.reward_signals: Dict[str, "RewardSignal"] = {}
```

Comment:
Used the string form here, because importing RewardSignal was leading to circular imports.